modules/services/views/src/main/java/au/com/shiftyjelly/pocketcasts/views/multiselect/MultiSelectAction.kt

```diff
@@ -108,59 +26,185 @@ sealed class MultiSelectAction(
         "select_all"
     )
 
+    sealed class BaseEpisodeAction(
+        override val groupId: Int,
+        override val actionId: Int,
+        @StringRes override val title: Int,
+        @DrawableRes override val iconRes: Int,
+        override val analyticsValue: String,
+        override val isVisible: Boolean = true,
+    ) : MultiSelectAction(
```

Comment:
It feels a bit strange that the episodes and bookmarks actions are in the same file. I'm not sure how to fix it, maybe a MultiSelectBookmarkActions and MultiSelectEpisodeActions class. 🤔

Reply:
Separated classes in 7dde7d0